docs/reference/commandline/dockerd.md

```diff
@@ -101,6 +102,82 @@ type `dockerd`.
 
 To run the daemon with debug output, use `dockerd -D`.
 
+> **Note**: In Docker 1.13 and higher, enable experimental features by starting
+> `dockerd` with the `--experimental` flag. In earlier Docker versions, a
+> different build was required to enable experimental features.Usage:	dockerd COMMAND
+
+A self-sufficient runtime for containers.
+
+Options:
```

Comment:
It's not a copy-paste error. It was called out in the related GH issue that we didn't provide this information, so I tried to be explicit about it.